style.css

```diff
@@ -127,3 +178,267 @@ body.active {
   display: flex;
   width: 100%;
 }
+
+.spacebetween {
+  padding: 57px 0;
+  background-color: #e5e5e5;
+}
+
+/* ---------------------------------------------------- work section css ------------------------------------------- */
+.workbackround {
+  background-color: #e5e5e5;
+}
+
+.row {
+  width: 100%;
+  padding: 12px 24px;
+}
+
+/* Remove extra left and right margins, due to padding */
+.column {
+  margin: -5px -8px;
+  background-color: #e5e5e5;
+}
+
+/* Clear floats after the columns */
+.column::after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+/* Style the works cards */
+.card {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.workimage {
+  padding: 16px 24px;
+}
+
+.innerpara {
+  width: 295px;
+  height: 76px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.back {
+  display: inline-block;
+  color: #6070ff;
+  text-align: left;
+  text-decoration: none;
+  font-size: 12px;
+  justify-content: space-between;
+  background-color: #ebebff;
+  padding: 5px;
+  border-radius: 8px;
+}
+
+.tonic {
+  text-align: left;
+  font-size: 32px;
+  color: #172b4d;
+}
+
+.button {
+  border: none;
+  color: #396df2;
+  padding: 16px 16px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 17px;
+  line-height: 24px;
+  font-weight: 500;
+  margin: 4px 2px;
+  transition-duration: 0.4s;
+  cursor: pointer;
+}
+
+.button:hover {
+  background-color: #008cba;
```

Comment:
- [ ] As the previous reviewer mentioned using hover color for the button, you did great work but you use a different color. This is a color code for hover `#6070FF`